src/tests/unit/cpu/ngraph_transformations/convert_to_interaction.cpp

```diff
@@ -0,0 +1,113 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <string>
+#include <memory>
+
+#include <ngraph/function.hpp>
+#include <ngraph/opsets/opset1.hpp>
+#include <ngraph/opsets/opset8.hpp>
+#include <ngraph_transformations/op/interaction.hpp>
+#include <ngraph_transformations/convert_to_interaction.hpp>
+#include <transformations/common_optimizations/nop_elimination.hpp>
+#include <transformations/smart_reshape/matmul_sr.hpp>
+#include <transformations/init_node_info.hpp>
+#include <transformations/utils/utils.hpp>
+#include <ngraph/pass/manager.hpp>
+#include <ie_core.hpp>
+
+#include "common_test_utils/ngraph_test_utils.hpp"
+
+using namespace testing;
+using namespace ov::intel_cpu;
+
+TEST(TransformationTests, ConvertToInteractionTest1) {
+    std::shared_ptr<ngraph::Function> f(nullptr), f_ref(nullptr);
+    {
+        using namespace ngraph;
```

Comment:
Please replace 'ngraph' to 'ov' where it is possible.

Reply:
remove ngraph in 9cb84c79a0c05b5517ae6fa4c744c62b0a629dfd